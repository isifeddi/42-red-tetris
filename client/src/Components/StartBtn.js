import React from "react";
import { StyledStartBtn } from "./styling/StyledStartBtn";
import { socket } from "../hooks";

const StartBtn = ({ callback, restart }) => (
  <StyledStartBtn onClick={callback}>{restart ? 'Restart Game' : 'Start the Game'}</StyledStartBtn>
);

export default StartBtn;
