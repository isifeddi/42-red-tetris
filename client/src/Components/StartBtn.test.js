import { render } from "@testing-library/react";
import StartBtn from "./StartBtn";

test("StartBtn render test ", () => {
  render(<StartBtn />);
});
