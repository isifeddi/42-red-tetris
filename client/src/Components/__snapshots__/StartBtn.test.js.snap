// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartBtn render test  1`] = `
<button
  className="sc-bdfBwQ hybXRk"
>
  Start Game
</button>
`;
